app/scripts/templates/sign_up.mustache

```diff
@@ -44,7 +44,7 @@
         <label for="show-password" class="show-password-label">
           {{#t}}Show{{/t}}
         </label>
-        <div class="input-help input-help-focused input-help-signup">{{#t}}Must be at least 8 characters{{/t}}</div>
+        <div class="input-help input-help-focused input-help-signup">{{#t}}A strong, unique password will keep your Firefox data safe from intruders. <a href="" tabindex="-1">More info.</a>{{/t}}</div>
```

Comment:
- link should be generated externally to minimize translations, see how we translate other links 
- link will not work in iOS and Android contexts. should it be hidden?